6.X/6.2.1/01-command_line_arguments.md

```diff
@@ -0,0 +1,192 @@
+<!-- Space: TOOLS -->
+<!-- Parent: Installer -->
+<!-- Title: Command Line Arguments -->
+<!-- Layout: plain -->
```

Comment:
@pravin-jothinathan Interesting ;-)
I think we have to live for now with this, but we need some automation.